exercises/1-the-manual-menace/README.md

```diff
@@ -143,53 +123,53 @@ NOTE - YAML is indentation sensitive so keep things lined up properly!
 
 5. Open the `inventory/host_vars/projects-and-policies.yml` file; you should see some variables setup already to create the `<YOUR_NAME>-ci-cd` namespace. This object is passed to the OpenShift Applier to call the `templates/project-requests.yml` template with the `params/project-requests-ci-cd` parameters. We will add some additional content here but first let's explore the parameters and the template
 
-6. Open the `params/project-requests-ci-cd` and replace the `<YOUR_NAME>` with your name to create the corresponding projects in the cluster.
-![new-item](../images/exercise1/ci-cd-project-namespace.png)
-
-7. Let's add two more params files to pass to our template to be able to create a `dev` and `test` project.
-  * Create another two params files `params/project-requests-dev` & `params/project-requests-test`. On the terminal run
-```bash
-touch params/project-requests-dev params/project-requests-test
-```
-  * In your editor; Open `params/project-requests-dev` and add the following by substituting `<YOUR_NAME>` accordingly
-```
-NAMESPACE=<YOUR_NAME>-dev
-NAMESPACE_DISPLAY_NAME=<YOUR_NAME> Dev
+6a. Inside of the `inventory/host_vars/projects-and-policies.yml` you'll see the following
+```yaml
+ci_cd:
+  NAMESPACE: "{{ namespace_prefix }}-ci-cd"
+  NAMESPACE_DISPLAY_NAME: "{{ namespace_prefix}}s CI/CD"
 ```
-  * In your editor; Open `params/project-requests-test` and add the following by substituting `<YOUR_NAME>` accordingly
+* This will define the variables that we'll soon be using to deploy our CI/CD project. It relies on the `namespace_prefix` that we updated earlier. Pulling these two sets of variables together will now allow us to pass the newly created variables to our template that will create our project appropriately. You'll notice that the name of the variable above (`ci_cd`) is then assigned to `params_from_vars` in our inventory.
+```yaml
+ansible_connection: local
+openshift_cluster_content:
+- object: projectrequest
+  content:
+  - name: "{{ ci_cd_namespace }}"
+    template: "{{ playbook_dir }}/templates/project-requests.yml"
+    action: create
+    params_from_vars: "{{ ci_cd }}"
+    tags:
+    - projects
 ```
-NAMESPACE=<YOUR_NAME>-test
-NAMESPACE_DISPLAY_NAME=<YOUR_NAME> Test
+
+7. Let's add two more params dicts to pass to our template to be able to create a `dev` and `test` project.At the top of `inventory/host_vars/projects-and-policies.yml` create a dictionary called `dev` and `test` similar to how you see `ci_cd` defined.
+  * In your editor; Open `inventory/host_vars/projects-and-policies.yml` and add the following:
+```yaml
+dev:
+  NAMESPACE: "{{ namespace_prefix }}-dev"
+  NAMESPACE_DISPLAY_NAME: "{{ namespace_prefix }} Dev"
+
+test:
+  NAMESPACE: "{{ namespace_prefix }}-test"
+  NAMESPACE_DISPLAY_NAME: "{{ namespace_prefix }} Test"
```

Comment:
Use the `title` filter here (?) to ensure uppercase first letter

Reply:
Updated all params_from_vars to include the `title` filter for NAMESPACE_DISPLAY_NAME